redbot/cogs/streams/streams.py

```diff
@@ -92,16 +104,62 @@ def check_name_or_id(self, data: str):
                 await self.bot.set_shared_api_tokens("twitch", client_id=token)
         await self.db.tokens.clear()
 
+    async def get_twitch_bearer_token(self) -> None:
+        tokens = await self.bot.get_shared_api_tokens("twitch")
+        if tokens.get("client_id"):
+            try:
+                tokens["client_secret"]
+            except KeyError:
+                prefix = (await self.bot._config.prefix())[0]
+                message = _(
+                    "You need a client secret key to use correctly Twitch API on this cog.\n"
+                    "Follow these steps:\n"
+                    "1. Go to this page: https://dev.twitch.tv/console/apps.\n"
+                    '2. Click "Manage" on your application.\n'
+                    '3. Click on "New secret".\n'
+                    "5. Copy your client ID and your client secret into:\n"
+                    "6. Run the command `{prefix}set api twitch client_id <your_client_id_here> "
```

Comment:
This should be changed to 
```suggestion
                    "`{prefix}set api twitch client_id <your_client_id_here> "
```

Reply:
Done.